src/NavWrapper/ProvideFeedback.tsx

```diff
@@ -10,14 +11,16 @@ export const ProvideFeedback: React.FC<any> = () => {
     const [submitting, setSubmitting] = useState(false);
     const [enabled, setEnabled] = useState(true);
     const [validated, setValidated] = useState(false);
+    const history = useHistory();
 
     const submitFeedback = async () => {
         try {
             setMessage('');
             setSubmitting(true);
             setEnabled(false);
+            console.log(history);
             await AxiosRequest.post('/support', {
-                description,
+                description: `${description} (Sent from: ${window.location.origin}${history.location.pathname})`,
```

Comment:
This should be a separate field and should be implemented on the backend
We might want to put this in a separate field in the future and I want all jira configurations on the backend

Reply:
Made a ticket for fixing this, we don't need to block merging for this, extra info is good and I want it for the release:
https://rederly.atlassian.net/browse/RED-124